### DISCUSSION TITLE
Procurement advice

### DISCUSSION THREAD

**Discussion by basak** [revk/xESP32-Faikout#43] [General]
Thank you for Faikin! I am in the market for a few air conditioning units that I can control by API. It looks like Faikin is about the best that can be managed! I'd like to get some Faikin-compatible units installed, and then order some of the boards from you. So a couple of questions then:

 1. Is there a list of Daikin models that Faikin is known to work with? Any chance you could start and publish a list, please? Given that I'll be buying these specifically to work with Faikin, I'd like to make sure what I buy will work, and sticking to a model that I know someone else has had success with would help with that.
 2. Are there any known installers that would be happy to work with a customer who wants to use Faikin immediately? Assuming not, is there any advice or prior experience from existing Faikin users please on whether or not to speak to the installer about Faikin, or just stick them in later? Again, maybe this might be worth documenting somewhere, along with a list (if it ever happens) of installers who are willing to work with Faikin directly.

Thanks!

**Comment by joshbenner** [comment]
My experience:
* Works with FTXS & CTXS models (specifically: FTXS12LVJU, FTXS09LVJU, and CTXS07LVJU). A wiki might be nice to collect some of this knowledge we accumulate?
* I used the provided PCB designs and got a bunch of these from JLCPCB (I'm not an electrical engineer, so this was a challenge -- but I _did_ manage to load the data into the site and get working units). I then flashed with TTL serial USB adapter.
* I doubt an installer would work directly with these devices; however, if your installer is willing, having them attach the cable while the unit is open could save some hassle. Some units of mine have come apart and gone back together more easily than others. See [another discussion](https://github.com/revk/ESP32-Faikin/discussions/39#discussioncomment-6070907) where I described how I made cables.

**Reply by revk** [reply]
It is a good idea to start a list though, I'll get on to that.

**Comment by revk** [comment]
https://github.com/revk/ESP32-Faikin/wiki/List-of-confirmed-working-air-con-units

**Reply by JeanOlivier** [reply]
I'm not sure how to properly contribute to this, I don't see an edit button and it's unclear to me where I'd do a PR.
But I'd like to add this model to the working list:

CTXS09HVJU

**Comment by revk** [comment]
What country are you in? I know a good UK installer, and they were impressed with what I had done, so I may talk to them about these. They will be a tad wary as not "official Diakin product", but may well be happy to sort leads and so on.

**Reply by basak** [reply]
I'm in the UK - North West. If they could install for me, even if entirely Faikin-free but understanding that this is what I intend to do afterwards, then I'd be very interested in speaking to them!

**Reply by revk** [reply]
I think they are mainly south, but not entirely sure. They coped with me messing with my own WiFi modules. I should have more next week so I may send them a couple.

**Comment by revk** [comment]
More stock at Amazon in a week ish.
